### PR TITLE
meral  file uploaded

### PR DESCRIPTION
**Pull request by meral-das**
lets gooo

### CODE REVIEW

File: src/Room3/Meral2.java [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package Room3;
+
+public class Meral2 {
```
Comment (MonikaSturman) [inline src/Room3/Meral2.java:3]: Please add a main method sweetie :)

Comment (meral-das) [reply src/Room3/Meral2.java:3]: Thank you for your review

Comment (AshwiniDhaneppagol) [reply src/Room3/Meral2.java:3]: It looks good

Comment (yavuzvelioglu) [reply src/Room3/Meral2.java:3]: @meral-das, i accepted

**Review by yavuzvelioglu — APPROVED** [review]

**Review by AshwiniDhaneppagol — APPROVED** [review]

**Pull request merged**